### ISSUE TITLE
Layout of table with hosts in configuration editor is not good

### ISSUE THREAD

**Issue by steven-de-wit** [backuppc/backuppc#296]
https://github.com/backuppc/backuppc/blob/9919bc17af3b58472ea85d81337e0471a6121deb/lib/BackupPC/CGI/EditConfig.pm#L1119

First time doing anything on Github, so please apologise if I did anything wrong.
Just looking to contribute by suggesting a fix for an issue I noticed in BackupPC.
Really like the product, this was just a visual issue in the CGI part.
Line 1119 needs to start a new table row, otherwise all hosts come out in a single row

**Comment by steven-de-wit** [comment]
Suggest to change line 1119 to
`<tr><td class="border">`

**Comment by moisseev** [comment]
@steven-de-wit You can open a `pull request` with the fix if you like.

I am just wondering what browser are you using? I thought modern browsers should be smart enough to render pages with such errors. That is why no one else has noticed this bug yet.

**Comment by steven-de-wit** [comment]
Hi @moisseev,

Thanks for the feedback.
Actually I'm using Chrome on Windows 10, just following regular updates (i.e. 77.0.3865.120 (Official Build) (64-bit)).
Since you asked, I reverted my local change and also tested with Chrome on Android8 (77.0.3865.116) as well as Firefox on Android8 (68.1.1) MS Edge on Win10 (44.18362.387.0), Chrome on Win8.1 (77.0.3865.120 (Official build) (32-bits)).
All the same behaviour.

FYI: I think they actually render it correctly... HTML doesn't instruct to start a new table row, so the browsers don't render it.

I've noticed this bug since years already (version 3.x and perhaps even before, can't remember when I started using BackupPC).
I mainly use the web-interface and I've always corrected the code myself so far.
Just upgraded from v3.3.1 to v4.3.1 today, had to put in a little bit more effort than usual as I'm running on Gentoo and the package isn't (well) maintained anymore over there.
Since I was putting in the effort (doing the deployment outside of package manager), I thought I'd put in the effort of letting the developers know about this (minor visual) bug.

I guess there's not a lot of people using the CGI-interface? :)

I'll figure out how to do a pull request, thanks.

Regards,
Steven
